libs/widget-lib/demo/vanilla.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>CoWSwap Widget demo</title>
+  <script src="https://cdn.jsdelivr.net/npm/@cowprotocol/widget-lib@0.1.2/index.iife.js"></script>
+</head>
+<body>
+  <div id="app"></div>
+  <script>
+    cowSwapWidget.cowSwapWidget({
+      container: document.getElementById("app"),
+      width: 400,
+      height: 600,
+      metaData: {
+        appKey: 'test',
+        appUrl: 'test',
+      }
+    }, {
+      env: 'dev'
```

Comment:
Is this a fixed list of envs?
Otherwise, how about we call it `demo`?

Reply:
>Is this a fixed list of envs?
Yes, only `'local' | 'prod' | 'dev' | 'pr'`.
We actually can keep it empty. I set `dev` only because we haven't deployed the widget changes on prod yet.

Reply:
No worries then